src/mongo/db/curop.cpp

```diff
@@ -446,6 +446,8 @@ uint64_t CurOp::MaxTimeTracker::getRemainingMicros() const {
     return _targetEpochMicros - now;
 }
 
+PseudoRandom CurOp::_prng(std::unique_ptr<SecureRandom>(SecureRandom::create())->nextInt64());
```

Comment:
I think using `time(0)` is a good old way, but if you want a proof, please see:
https://github.com/percona/percona-server-mongodb/blob/v3.2/src/mongo/s/chunk.cpp#L197